projects/aca-playwright-shared/src/api/shared-links-api.ts

```diff
@@ -48,4 +50,50 @@ export class SharedLinksApi {
       return null;
     }
   }
+
+  async shareFilesByIds(ids: string[]): Promise<SharedLinkEntry[]> {
+    const sharedLinks: SharedLinkEntry[] = [];
+    try {
+      if (ids && ids.length > 0) {
+        for (const id of ids) {
+          const sharedLink = await this.shareFileById(id);
+          sharedLinks.push(sharedLink);
+        }
+      }
+    } catch (error) {
+      logger.error(`SharedLinksApi shareFilesByIds : catch : `, error);
+    }
+    return sharedLinks;
+  }
+
+  private async getSharedLinks(maxItems: number = 250): Promise<SharedLinkPaging | null> {
+    try {
+      const opts = {
+        maxItems
+      };
+      return await this.apiService.share.listSharedLinks(opts);
+    } catch (error) {
+      logger.error(`SharedLinksApi getSharedLinks : catch : `, error);
+      return null;
+    }
+  }
+
+  async waitForFilesToBeShared(filesIds: string[]): Promise<any> {
```

Comment:
If this promise shouldn't return anything you can use `Promise<void>`

Reply:
thanks, added